flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/source/proxy/FullJitterBackoff.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.dynamodb.source.proxy;
+
+import org.apache.flink.annotation.Internal;
+
+import java.util.Random;
+
+/**
+ * Used to calculate full jitter backoff sleep durations.
+ *
+ * @see <a href="https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/">
+ *     Exponential Backoff and Jitter </a>
+ */
+@Internal
+public class FullJitterBackoff {
+
+    /** Random seed used to calculate backoff jitter for DynamoDb Streams operations. */
+    private final Random seed = new Random();
+
+    /**
+     * Calculates the sleep time for full jitter based on the given parameters.
+     *
+     * @param baseMillis the base backoff time in milliseconds
+     * @param maxMillis the maximum backoff time in milliseconds
```

Comment:
Why do we need this? It doesn't seem used

Reply:
In either case, could we consider using the AWS SDK retry policy instead?

Reply:
Yes, we'll be using AWS SDK retry policy. This missed cleanup